layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,206 @@
+      <li>Michael Brown</li>
+      <li>Michael Dean</li>
+      <li>Mike Ramos</li>
+      <li>Miles Hall</li>
+      <li>Natosha McDade</li>
+      <li>Patrick Harmon</li>
+      <li>Philando Castile</li>
+      <li>Quintonio LeGrier</li>
+      <li>Ramarley Graham</li>
+      <li>Robert Lawrence White</li>
+      <li>Rumain Brisbon</li>
+      <li>Ryan Twyman</li>
+      <li>Samuel David Mallard</li>
+      <li>Samuel DuBose</li>
+      <li>Sandra Bland</li>
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> names not listed here whose lives were taken by violence and brutality</li>
+    </ul>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you.</p>
+
+    <!-- <section id="we-are-here-for-you">Tierney Cyren (<a href='https://twitter.com/bitandbang/status/1267893948149911556?s=20'>message of support</a>, <a href='https://twitter.com/bitandbang'>twitter</a>), Joe Sepi (<a href='https://twitter.com/joe_sepi/status/1268249157317332993'>message of support</a>, <a href='https://twitter.com/joe_sepi'>twitter</a></section> -->
```

Comment:
Wait, why is this commented out?

Reply:
Was requested to be commented out until more names were added. Again @nodejs/tsc and @nodejs/community-committee please add your names here if possible <3

Reply:
Commented out until more statements of support from across the project are available.

Reply:
Would this perhaps be a bit easier to add to as an unordered list?

Reply:
@MylesBorins my only concern there is putting weight on us at the same weight as the list of people above. @joesepi shared a bit of CSS we could potentially add that would solve for this as a ul but make it inline, I can grab when I switch computers.

Reply:
@MylesBorins here is the message I sent to  @bnb earlier:

“
I wonder if the names should be inline rather than on each line. Like they did on Ember site. I tested this by making `ul.humans > li { display: inline; }` and adding `ul.humans > li::after { content: ", " }` and it looked better to me
“

Reply:
I've pushed this change up fwiw

Reply:
![image](https://user-images.githubusercontent.com/502396/83704696-d8fdf200-a5e0-11ea-81b4-8a653af32244.png)

Reply:
![image](https://user-images.githubusercontent.com/502396/83704839-43169700-a5e1-11ea-8a1c-65f3ad67cb21.png)